DEPS

```diff
@@ -91,6 +91,113 @@ vars = {
 
   # Setup Git hooks by default.
   "setup_githooks": True,
+
+  # upstream URLs for third party dependencies, used in
+  # determining common ancestor commit for vulnerability scanning
+  # prefixed with "upstream_" in order to be identified by parsing tool
+  "upstream_zlib": "https://github.com/madler/zlib.git",
```

Comment:
Can this list be alphabetized by the string on the left side of the colon, please?

Reply:
Done